### DISCUSSION TITLE
Use of discussions

### DISCUSSION THREAD

**Discussion by danlangford** [pappde/bmai#3] [General]
Discussions is a fairly new feature to GitHub but I think it will be valuable. I plan to drop questions here about the code base and use the space to find track my search for the answers. I don’t want you @hamstercrack to feel like you must spend time finding the answer but if you happen to know the answer or some info off the top of your head you could maybe quickly mention it. It will be a way for me to track my outstanding questions and as I find the answers I will report back here also. I think it’s nice because it’s a more open forum and you don’t need to be swamped with emails and feel obligated to reply quickly.

**Comment by pappde** [comment]
Absolutely - I'm sure I will enjoy answering any questions.
